craft_providers/lxd/launcher.py

```diff
@@ -36,6 +38,32 @@
 logger = logging.getLogger(__name__)
 
 
+class InstanceTimer(threading.Thread):
+    """Timer for update instance that still alive."""
+
+    count: int = 0
+    active: bool = True
+    instance: LXDInstance
+
+    def __init__(self, instance: LXDInstance) -> None:
+        self.instance = instance
+        threading.Thread.__init__(self, daemon=True)
+
+    def run(self) -> None:
+        """Run the timer."""
+        while self.active:
+            self.count += 1
+            self.instance.config_set("user.craft_providers.timer", str(self.count))
+            logger.debug("Set instance timer to %r", self.count)
+            time.sleep(3)
```

Comment:
Perhaps we should have the tick period be configurable in `__init__`?

Reply:
Added